src/styles/blocks/header.scss

```diff
@@ -0,0 +1,95 @@
+.header {
+  position: sticky;
+  z-index: 2;
+  top: 0;
+  left: 0;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  background-color: $color-white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+
+  &__logo {
+    position: relative;
+    width: 40px;
+    min-width: 40px;
+    height: 40px;
+    margin: 10px 0;
+    transition: all 0.3s;
```

Comment:
Transition all is not good for performance. Better specify the exact properties.
Also, transition duration should be the same everywhere